test/xpu_api/numerics/numeric.ops/inner.product/xpu_inner_product.pass.cpp

```diff
@@ -96,7 +94,10 @@ class KernelTest23;
 class KernelTest24;
 class KernelTest25;
 
-int main() {
+int
+main()
+{
+#if 0
```

Comment:
I suggest guarding some test cases with a macro which disables them when debug mode is used, e.g. `PSTL_USE_DEBUG`